crates/cast/bin/opts.rs

```diff
@@ -625,6 +625,10 @@ pub enum Subcommands {
 
         #[clap(flatten)]
         rpc: RpcOpts,
+
+        /// Token address to query, with the method `balanceOf(address) return (uint256)`
+        #[clap(short, long)]
+        token: Option<Address>,
```

Comment:
ERC721 is also a token, so this is ambiguous. Maybe `--erc20`?

Reply:
The ERC721 Standard do implement the same method `balanceOf(address) returns (uint256)`. IMHO, we don't need to separate them if we do the same call. Anyway I just want to simplify the call. 
- https://eips.ethereum.org/EIPS/eip-721 

It's ok for me to implement `--erc20` and `--erc721` options. And I will add some tests.

Reply:
That's true, but [ERC1155](https://eips.ethereum.org/EIPS/eip-1155) has a different signature

you can alias --erc721 to --erc20 since it will have the same effect.

Reply:
Ok. Appreciated for the guide.